at_secondary/at_secondary_server/lib/src/server/at_secondary_impl.dart

```diff
@@ -421,6 +421,8 @@ class AtSecondaryServerImpl implements AtSecondaryServer {
     hiveKeyStore.commitLog = atCommitLog;
     _commitLog = atCommitLog;
     keyStoreManager.keyStore = hiveKeyStore;
+    // Initialize the hive metadata cache
+    hiveKeyStore.initMetaDataCache();
```

Comment:
This need not be called explicitly. Can you check how commit log cache is initialised?

Reply:
The commit log Keystore already had an initialize method prior to introducing the cache. The cache is initialized through that method.